parse/grammar/KuneiformParser.g4

```diff
@@ -184,13 +216,58 @@ sql:
 
 sql_statement:
     (WITH RECURSIVE? common_table_expression (COMMA common_table_expression)*)?
-    (select_statement | update_statement | insert_statement | delete_statement)
+    (create_table_statement | alter_table_statement| create_index_statement | drop_index_statement // ddl
+    | select_statement | update_statement | insert_statement | delete_statement) // dml
 ;
 
 common_table_expression:
     identifier (LPAREN (identifier (COMMA identifier)*)? RPAREN)? AS LPAREN select_statement RPAREN
 ;
 
+create_table_statement:
+    CREATE TABLE name=identifier
+    LPAREN
+    (c_column_def | constraint_def | c_index_def)
+    (COMMA  (c_column_def | constraint_def | c_index_def))*
+    RPAREN
+;
+
+constraint_def: // TODO: rename to table_constraint
+    (CONSTRAINT name=identifier)?
+    unnamed_constraint
+;
+
+unnamed_constraint:
+    PRIMARY KEY LPAREN identifier_list RPAREN
+    | UNIQUE LPAREN identifier_list RPAREN
+    | CHECK LPAREN sql_expr RPAREN
+    | FOREIGN KEY LPAREN identifier RPAREN fk_constraint
```

Comment:
not a huge deal but it might be easier if these are named? e.g.

```
unnamed_constraint:
    PRIMARY KEY LPAREN identifier_list RPAREN #unnamed_constraint_pk
    | UNIQUE LPAREN identifier_list RPAREN #unnamed_contract_unique
```

Reply:
the syntax of the constraint is simple enough, for now it's done just in one switch, I don't think it's necessary to create different tree nodes on constraints.